_posts/2018-04-11-dont-make-users-think-about-updating-software.md

```diff
@@ -49,35 +49,35 @@ We ran into the problem of people using outdated software with deprecated urls a
   <div style="width:50%;padding:1em" markdown="1">
 #### AGRC Extension Building v1.2.0
 
-To combat the problem of people not checking GitHub for updates, we updated our extensions to ping GitHub and notify the user to go and download the new version. This improved the user experience but the user interaction was still **unnacceptable**.
+To combat the problem of people not checking GitHub for updates, we updated our extensions to ping GitHub and notify users to go and download the new version. This improved the user experience but the user interaction was still **unacceptable**.
   </div>
 </div>
 
 # tl;dr
 ## AGRC Extension Building v2.0.0
 
-AGRC has created [Pro Evergreen](https://github.com/steveoh/pro-evergreen), an auto update utility for ArcGIS Pro add-ins. Every add-in that comes out of AGRC will use this package to keep the add-ins evergreen. Pro Evergreen contains functionality to compare the installed version of an add-in with the latest releases on GitHub. More importantly, the functionality to update an add-in is quite simple for the developer.
+AGRC has created [Pro Evergreen](https://github.com/steveoh/pro-evergreen), an auto update utility for ArcGIS Pro add-ins. Every add-in that comes out of AGRC will use this package to keep the add-ins evergreen. Pro Evergreen has the functionality to compare the installed version of an add-in with the latest releases on GitHub. More importantly, the functionality to update an add-in is quite simple for the developer.
 
 > evergreen.Update(newVersion);
 
-The developer has the choice of how or if they want to notify the user of the update. Updates can be made silently or a notification could be displayed; The UI is up to the add-in developer. Using GitHub release and pre-release assets, users can be put in "beta" or "stable" channels. This allows the developer to test changes before delivering to a larger audience. There are a lot of benefits to this approach and we hope you can benefit from our experience and efforts.
+The developer has the choice of how (and whether) they want to notify the user of the update. Updates can be made silently or a notification could be displayed; the UI is up to the add-in developer. Using GitHub release and pre-release assets, users can be put in "beta" or "stable" channels. This allows the developer to test changes before delivering to a larger audience. There are a lot of benefits to this approach, and we hope you can benefit from our experience and efforts.
 
 #### Getting Started
 
-The evergreen documentation is in the repository [read me](https://github.com/steveoh/pro-evergreen/blob/master/README.md) but getting started with Pro Evergreen is 10 steps or less.
+The evergreen documentation is in the repository [read me](https://github.com/steveoh/pro-evergreen/blob/master/README.md) and getting started with Pro Evergreen is 10 steps or fewer.
```

Comment:
Again, a little weird, but "fewer" is used with concrete/countable objects. This topic is a little subjective though, so take that for what you will!